examples/tests/valid/json.w

```diff
@@ -0,0 +1,16 @@
+let json_string  = Json "hello";
+let json_number  = Json 123;
+let json_bool    = Json true;
+let json_array   = Json [ 1, 2, 3 ];
+let json_obj     = Json { boom: 123 };
+
+let json_mut_obj = Json {
```

Comment:
Should it be `MutJson`?
```suggestion
let json_mut_obj = MutJson {
```

Reply:
Yupp it should, I committed some testing junk 😅